name-service/js/test/e2e/index.test.ts

```diff
@@ -0,0 +1,129 @@
+import {
+  Connection,
+  Keypair,
+  LAMPORTS_PER_SOL,
+  PublicKey,
+  sendAndConfirmTransaction,
+  Transaction,
+} from '@solana/web3.js';
+import chai, { expect } from 'chai';
+import chaiAsPromised from 'chai-as-promised';
+
+import {
+  createNameRegistry,
+  deleteNameRegistry,
+  getHashedName,
+  getNameAccountKey,
+  NameRegistryState,
+  reallocNameAccount,
+  transferNameOwnership,
+  updateNameRegistryData,
+} from '../../src';
+
+chai.use(chaiAsPromised);
+const url = 'http://localhost:8899';
+
+describe('Name Service Program', () => {
+  let connection: Connection;
+  let payer: Keypair;
+  let owner: Keypair;
+  let name: string;
+  let nameKey: PublicKey;
+  before(async () => {
+    connection = new Connection(url, 'confirmed');
+    payer = Keypair.generate();
+    const airdropSignature = await connection.requestAirdrop(
+      payer.publicKey,
+      4 * LAMPORTS_PER_SOL
+    );
+    await connection.confirmTransaction(airdropSignature, 'confirmed');
+    name = '.sol';
+    nameKey = await getNameKey(name);
```

Comment:
nit: It looks like these are always the same, so should we just keep them as consts declared higher up?